parsers/CO.py

```diff
@@ -74,9 +74,11 @@ def fetch_live_consumption(
     demand_list = TotalConsumptionList(logger)
 
     for datapoint in demand_data:
+        if ("." in datapoint["Fecha"]) and (len(datapoint["Fecha"].split(".")[1]) != 3):
+            datapoint["Fecha"] = datapoint["Fecha"].ljust(23, "0")
```

Comment:
I think adding a comment to what this does and why will help with future maintainability.